### PR TITLE
pretty assert

### PR DESCRIPTION
**Pull request by conorbros**
Seeing if there is interest in using this library. An example can be seen in the logs.

### CODE REVIEW

**Comment by rukai** [comment]
Woah, I love this!
Lets set it to a dev-dependency instead of regular dependency and land this!

**Comment by rukai** [comment]
Is the test failure intentional?

**Review by rukai — APPROVED** [review]
nice!

File: test-helpers/Cargo.toml [hunk 1]
```diff
@@ -39,3 +39,4 @@ http = "1.1.0"
 rustls = { version = "0.21.2" ,features = ["dangerous_configuration"] }
 rustls-pemfile = "1.0.2"
 tokio-tungstenite = { git = "https://github.com/shotover/tokio-tungstenite", features = ["rustls-tls-native-roots"], rev = "2f1cc11e491c2d0401d01a83f623623182784b3a" }
+pretty_assertions = "1.4.0"
```
Comment (rukai) [inline test-helpers/Cargo.toml:42]: may as well make this a dev-dependency

**Review by rukai — CHANGES_REQUESTED** [review]
noticed some issues

File: test-helpers/Cargo.toml (hunk 1 above)
Comment (conorbros) [reply test-helpers/Cargo.toml]: It's in the "test-helpers" module which means it's already only included as dev-dependency.

**Review by rukai — APPROVED** [review]

**Review by ronycsdu — APPROVED** [review]

**Pull request merged**